Modules/SubModule_Import/xASL_imp_NII2BIDS_Subject.m

```diff
@@ -50,7 +49,11 @@ function xASL_imp_NII2BIDS_Subject(imPar, bidsPar, studyPar, nameSubjectSession)
 function bidsLabel = xASL_imp_CheckForAliasInVisit(imPar,nameSubjectSession)
 
     % Get visitAliases from imPar
-    visitAliases = imPar.tokenVisitAliases(:,2);
+    if isfield(imPar,'tokenVisitAliases') && ~isempty(imPar.tokenVisitAliases) && size(imPar.tokenVisitAliases,2)>1
```

Comment:
You can omit the `~isempty` if you check for `(size( , 2)>1`

Reply:
Doesn't hurt though...

Reply:
it doesn't add anything. If it is empty, size()>1 will also return 0. So it is just extra time reading, would remove this.